gateway/src/shard/impl.rs

```diff
@@ -92,6 +99,10 @@ impl Error for CommandError {
 #[derive(Debug)]
 #[non_exhaustive]
 pub enum CommandErrorType {
+    /// Heartbeat has not been initialized yet.
+    HeartbeatNotActive,
```

Comment:
This verbiage feels a little unclear to me. Could we clarify it's talking about the heartbeater?

Reply:
Should be more explicit now